src/pybind/mgr/dashboard/services/access_control.py

```diff
@@ -194,12 +194,12 @@ def from_dict(cls, r_dict):
                     r_dict['scopes_permissions'])
 
     @classmethod
-    def map_to_system_roles(cls, roles) -> List['Role']:
+    def map_to_system_roles(cls, idp_roles: List[str]) -> List['Role']:
```

Comment:
NTI: this is not necessarly IDP related
```suggestion
    def map_to_system_roles(cls, roles: List[str]) -> List['Role']:
```

Reply:
I did this since in case of IdP only we are using this mapper `Role.map_to_system_roles` .
I cant find a flow that uses this for non IdP purposes.